pkg/operator/resources/crds_generated.go

```diff
@@ -4464,7 +4464,7 @@ status:
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.11.3
+    controller-gen.kubebuilder.io/version: v0.13.0
```

Comment:
I think your old volume from 1.57 or something is picking up a wrong `controller-gen` when you run make targets. It should definitely be v0.11.3 in this branch https://github.com/kubevirt/containerized-data-importer/blob/a45fd9fced250343656a6edd443ef25d47326d59/hack/build/docker/builder/Dockerfile#L49